docs/migration.md

```diff
@@ -3,14 +3,42 @@ title: Migration
 sort_rank: 10
 ---
 
+# Prometheus 3.0 migration guide
+
+In line with our [stability 
+promise](https://prometheus.io/docs/prometheus/latest/stability/),
+the Prometheus 3.0 release contains a number of backwards incompatible changes.
+This document offers guidance on migrating from Prometheus 2.x to Prometheus 3.0 and newer versions.
+
+## Flags
+
+- Some previously deprecated feature flags have been removed. Check that you no 
+  longer pass any of the following to `--enable-feature`: `promql-at-modifier`
+  `promql-negative-offset`, `remote-write-receiver`, 
+  `new-service-discovery-manager`, `expand-external-labels`, 
+  `no-default-scrape-port`.
```

Comment:
`agent` is also missing in this list. It was removed in https://github.com/prometheus/prometheus/pull/14747